### PR TITLE
Queue empty state

### PR DESCRIPTION
**Pull request by emily-plummer**
### Purpose
Adds a generic empty queue state :)
### Review
cc @hamstu 

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.publish.buffer.com :smile:_

### CODE REVIEW

File: packages/queue/components/QueuedPosts/index.jsx [hunk 1]
```diff
@@ -39,14 +41,20 @@ const QueuedPosts = ({
...
   return (
     <div>
       <div style={composerStyle}>
         <Input
           placeholder={'What would you like to share?'}
         />
       </div>
+      {total < 1 &&
+        <EmptyState
+          title={'It looks like you haven\'t got any posts in your queue!'}
```
Comment (hamstu) [inline packages/queue/components/QueuedPosts/index.jsx]: I wonder if we should use translations more? The same way @hharnisc did in `profile-sidebar`. Not a high priority one, since I have a lot of strings I've been meaning to move over too, but it would be cool to move towards that.

File: packages/shared-components/EmptyState/index.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    width: '100%',
+  };
+  return (
+    <div style={wrapperStyle}>
+      <div style={containerStyle}>
+        {heroImg &&
+          <img style={imageStyle} alt="" src={heroImg} />}
+        {title &&
+          <div style={headerStyle}>
+            <Text size={'large'} weight={'bold'}>
```
Comment (hamstu) [inline packages/shared-components/EmptyState/index.jsx]: What do you think about using `size="large"`, `weight="bold"`, etc. instead? I think it's a bit cleaner/easier to read, since you don't need the curly braces. I've been doing this in other places and I feel like it would be good to standardize, esp. since eslint doesn't care either way.

If it feels good to you, I'd say this would apply to the whole PR.

File: packages/shared-components/EmptyState/story.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import EmptyState from './index';
+
+
+storiesOf('EmptyState')
+  .addDecorator(checkA11y)
+  .add('default', () => (
+    <EmptyState
+      title={'It looks like you haven\'t got any posts in your queue!'}
+      subtitle={'Click the box above to add a post to your queue :)'}
+      heroImg={'https://s3.amazonaws.com/buffer-publish/images/fresh-queue.png'}
```
Comment (hamstu) [inline packages/shared-components/EmptyState/story.jsx]: So fresh! 🍓  ✨ 

One request; could you use this `@2x` version instead and then scale down to the normal size so it's not fuzzy on Retina?
https://www.dropbox.com/s/bwfvz2f49xpck55/fresh-queue%402x.png?dl=0

File: packages/queue/components/QueuedPosts/index.jsx (hunk 1 above)
Comment (hamstu) [inline packages/queue/components/QueuedPosts/index.jsx]: Maybe something we can do in a separate PR, time permitting.

File: packages/shared-components/EmptyState/story.jsx (hunk 3 above)
Comment (emily-plummer) [inline packages/shared-components/EmptyState/story.jsx]: yes! thank you!

File: packages/shared-components/EmptyState/index.jsx (hunk 2 above)
Comment (emily-plummer) [inline packages/shared-components/EmptyState/index.jsx]: will do!

File: packages/queue/components/QueuedPosts/index.jsx (hunk 1 above)
Comment (emily-plummer) [inline packages/queue/components/QueuedPosts/index.jsx]: Agreed!
